junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/AbstractJreRangeCondition.java

```diff
@@ -71,8 +71,8 @@ protected final boolean isCurrentVersionWithinRange(JRE minJre, JRE maxJre, int
 		Preconditions.condition((min != JRE.MINIMUM_VERSION || max != Integer.MAX_VALUE),
 			() -> "You must declare a non-default value for the minimum or maximum value in @" + this.annotationName);
 		Preconditions.condition(min >= JRE.MINIMUM_VERSION,
-			() -> String.format("@%s's minimum value [%d] must greater than or equal to %d", this.annotationName, min,
-				JRE.MINIMUM_VERSION));
+			() -> String.format("@%s's minimum value [%d] must be greater than or equal to %d", this.annotationName,
```

Comment:
Oops. Thanks for adding the missing word. 👍 

Though, it looks like we didn't have a test to catch that -- yes/no? 🤔

Reply:
Looks like we don't: https://app.codecov.io/gh/junit-team/junit5/blob/main/junit-jupiter-api%2Fsrc%2Fmain%2Fjava%2Forg%2Fjunit%2Fjupiter%2Fapi%2Fcondition%2FAbstractJreRangeCondition.java#L74

I'll add one

Reply:
I've removed this validation for 5.13.0-RC1 in https://github.com/junit-team/junit5/pull/4517 as it seems unreachable. See rationale in [`8b8101b` (#4517)](https://github.com/junit-team/junit5/pull/4517/commits/8b8101bf69ef93c870865ddbb035f19d4f1740a4)